packages/react-storage/src/components/StorageBrowser/views/LocationActionView/UploadControls.tsx

```diff
@@ -329,14 +338,7 @@ export const UploadControls = (): JSX.Element => {
       />
       <Exit onClick={() => handleUpdateState({ type: 'CLEAR' })} />
       <Title />
-      <Primary
-        disabled={disablePrimary}
-        onClick={() => {
-          handleUpload();
-        }}
-      >
-        Start
-      </Primary>
+      <ActionStartControl className={`${CLASS_BASE}__primary`} />
```

Comment:
Here, where the control is actually used, we should give it somewhat unique class names to its particular "usage" within a view. For example, since it's used here in the upload view...
```suggestion
      <ActionStartControl className={`${CLASS_BASE}__upload-action-start`} />
```